docs/technical_reference/unit_models/zero_order_unit_models/anaerobic_mbr_mec_zo.rst

```diff
@@ -8,12 +8,12 @@ See documentation for :ref:`reactive single-inlet, double-outlet Helper Methods<
 
 Electricity Consumption
 -----------------------
-Electricity consumption is calculated using the constant_intensity helper function.
+Electricity consumption is calculated using the pump_electricity helper function.
 See documentation for :ref:`Helper Methods for Electricity Demand<electricity_methods>`.
 
 Costing Method
 --------------
-Costing is calculated using the False method in the zero-order costing package.
+Costing is calculated using the cost_anaerobic_mbr_mec method in the zero-order costing package.
```

Comment:
```suggestion
Costing is calculated using the **cost_anaerobic_mbr_mec** method in the zero-order costing package.
```

Reply:
Added this too in the automate script (also bolding the "model type")